uiowa/uiowa
# [ITS] Alerts Calendar

Resolves #7234 
Relates to #7360 





# How to test


```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli
```

Edit an existing alert and/or add a new alert
Check that the Title field is not available

Edit a page and add the `Alerts calendar` block
Add some alerts of different types, check them out on the calendar, and see how it works!
Check both the `Month` and `List` displays
Load the page on a smaller screen, and see that it defaults to `List`
Load the page on a bigger screen, and see that it defaults to `Month`

Any suggestions on the 'Legend' part of this would be great as well

## Updates

The style adjustments have moved to uids_base, so a `blt frontend` will be needed

In addition to the calendar stuff specifically, check https://its.uiowa.ddev.site/alerts and see that the badges for the 4 types of alerts match the colors in the calendar (ie planned maintenance and service announcement badge colors have been updated)
